core/domain/learner_progress_services.py

```diff
@@ -311,6 +314,27 @@ def remove_exp_from_completed_list(user_id, exploration_id):
             save_completed_activities(activities_completed)
 
 
+def remove_exp_ids_from_completed_list(user_id, exploration_ids):
```

Comment:
Why don't you combine this with remove_collection_ids_from_completed_list, so that you only need to retrieve the CompletedActivitiesModel once?